cfme/ansible/repositories.py

```diff
@@ -0,0 +1,237 @@
+# -*- coding: utf-8 -*-
+"""Page model for Automation/Anisble/Repositories"""
+from navmazing import NavigateToAttribute, NavigateToSibling
+
+from cfme.base import Server
+from cfme.exceptions import ItemNotFound
+from cfme.base.login import BaseLoggedInPage
+from widgetastic.widget import Text, Checkbox
+from widgetastic_manageiq import Table
+from widgetastic_patternfly import Dropdown, Button, Input, FlashMessages
+from utils.appliance import Navigatable
+from utils.appliance.implementations.ui import navigator, navigate_to, CFMENavigateStep
+from utils.wait import wait_for
+
+
+class RepositoryBaseView(BaseLoggedInPage):
+    flash = FlashMessages('.//div[starts-with(@class, "flash_text_div") or @id="flash_text_div"]')
+    title = Text(locator=".//h1")
```

Comment:
REQUIRED: this is too broad locator. I'd advice you to use locator like 
`title = Text('//div[@id="main-content"]//h1')`